addon-test-support/qunit/test-runner.js

```diff
@@ -36,7 +36,7 @@ export default function testFeature(feature, yadda, yaddaAnnotations, library) {
 function testScenario(scenario, feature, yadda, yaddaAnnotations, library) {
   let scenarioAction = yaddaAnnotations.runScenario(feature.annotations, scenario.annotations);
   if (typeof scenarioAction === 'function') {
-    scenarioAction.call(this, scenario);
+    scenarioAction.apply(this, [].slice.call(arguments));
```

Comment:
Why the slicing here, and not just `scenarioAction.apply(this, arguments)`?

Reply:
You're right - the slicing isn't necessary, it should just be scenarioAction.apply(this, arguments)